torch/_inductor/decomposition.py

```diff
@@ -32,6 +32,15 @@
         aten.tril_indices,
         aten.triu_indices,
         aten.unsafe_split,
+        aten.isneginf,
```

Comment:
can you add them to `_decomp/__init__.py` instead?

Reply:
Sure.  Actually turns out most of them were already there.  Not sure why @fdrocha needed to add `make_fallback`s for them.